rllib/utils/exploration/tests/test_explorations.py

```diff
@@ -85,7 +85,7 @@ class TestExplorations(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        ray.init(num_cpus=4)
+        ray.init()
```

Comment:
We shouldn't call `ray.init()` with any extra parameter in unittests. especially if we want to run these tests on an existing clusters.